Sources/TuistGraph/DependenciesGraph/DependenciesGraph.swift

```diff
@@ -1,11 +1,11 @@
 import Foundation
 import TSCBasic
+import TuistSupport
```

Comment:
I see that this `import` is needed for `FatalError`. @fortmarek can you verify if it is ok to import `TuistSupport` in `TuistGraph`? I guess it is but Id like to keep a hand on pulse. 😅

Reply:
This does not go against the overall architecture, although `TusitGraph` should be as simple as possible. However, I believe for this use case it's fine. We might, however, want to extract the business logic traverser - as it's done for the regular graph. But I'd reconsider that again before merging to main once we have a clearer picture about this component.